lib/services/importPersonas/personasImportHelpers.js

```diff
@@ -137,9 +139,35 @@ export const getPersonaName = ({
   row
 }) => {
   const nameColumn = chain(structure)
-    .findKey(item => item.columnType === COLUMN_NAME);
+    .findKey(item => item.columnType === COLUMN_NAME)
+    .value();
+
+  if (nameColumn !== undefined) {
+    return row[nameColumn];
+  }
+
+  const firstNameColumn = chain(structure)
+    .findKey(item => item.columnType === COLUMN_FIRST_NAME)
+    .value();
+
+  const lastNameColumn = chain(structure)
+    .findKey(item => item.columnType === COLUMN_LAST_NAME)
+    .value();
 
-  return row[nameColumn];
+  const firstName = row[firstNameColumn];
```

Comment:
trim(`${get(row, firstNameColumn)} ${get(row, lastNameColumn)}`);

Maybe a little more concise.

Reply:
Nice.
If a first name starts with spaces, e.g. "  abc", we should leave them, shouldn't we?
@cbishopvelti @ryansmith94

Reply:
If trimming is ok, I'll refactor this function.

Reply:
I don't have any issues with trim except for the inconsistency.

Let's say one row has a first name of "a " and no last name, their full name would be "a". However, another row could have the same first name with a last name of "b" and their full name would be "a  b" (two spaces between). You could solve this issue with ```trim(`${trim(firstName)} ${trim(lastName)}`)```.

Note that this returns an empty string when no firstName or lastName are provided, so perhaps we should check for that and return undefined in that case to maintain the current functionality.

Reply:
OK